src/core/Global.ttslua

```diff
@@ -2134,7 +2136,11 @@ function TokenManager.spawnMultipleTokens(card, tokenType, tokenCount, shiftDown
     local matColor = playermatApi.getMatColorByPosition(card.getPosition())
     local activeInvestigatorData = playermatApi.getActiveInvestigatorData(matColor)
     callbackName = "updateUniversalActionAbilityToken"
-    callbackParams = { class = activeInvestigatorData.class, symbol = subType or activeInvestigatorData.class }
+    if temporary then
```

Comment:
Formatting:
Add an empty line before an `if` statement (unless the previous line is directly connected to it)